public/controllers/management/components/management/ruleset/ruleset-editor.js

```diff
@@ -277,8 +308,8 @@ class WzRulesetEditor extends Component {
                         width="100%"
                         height={`calc(100vh - ${((showWarningRestart && !xmlError) || wazuhNotReadyYet) ? 300 : (xmlError ? (!showWarningRestart ? 245 : 350) : 230)}px)`}
                         value={content}
-                        onChange={newContent =>
-                          this.setState({ content: newContent })
+                        onChange={newContent =>{
+                          this.setState({ content: newContent })}
```

Comment:
A good practice for this type of event is to separate it, create a new external event.
`onChange={myNewMethodOnChange}
`
.......

```
  const myNewMethodOnChange = (newContent) => {
    this.setState({ content: newContent });
  };
```